requirements_test_all.txt

```diff
@@ -221,6 +221,9 @@ coinmarketcap==5.0.3
 # homeassistant.scripts.check_config
 colorlog==4.2.1
 
+# homeassistant.components.upc_connect
+connect-box==0.2.7
```

Comment:
huh?

Reply:
The pre-commit hook keeps adding this. It is unrelated and hopefully goes away after rebasing this onto dev once #39305 is merged.